packages/kit/types/index.d.ts

```diff
@@ -67,6 +67,7 @@ export interface Builder {
 	getClientDirectory(): string;
 	getServerDirectory(): string;
 	getStaticDirectory(): string;
+	getPrefixedAppDirectory(): string;
```

Comment:
I'm not sure "directory" is the right term here. Probably "path" is more applicable. I think we can drop "prefixed from the name and just add some docs instead:
```suggestion
  /** The URL path for the app directory including any configured base path */
	getAppPath(): string;
```